config.fnl

```diff
@@ -326,22 +345,24 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
 (local menu-items
-       [{:key :space
-         :title "Alfred"
+       [{:key    :space
+         :title  "Alfred"
          :action (activator "Alfred 4")}
-        {:key :w
+        {:key   :w
          :title "Window"
+         :enter (fn [] (windows.show-display-numbers))
```

Comment:
The first argument this function receives is the window-menu table which includes the title, key, and window-bindings as :items. 

I would recommend moving the logic to add items for each window here. This way if the user changes monitors or moves around they don't have to manually restart hammerspoon to show the correct monitor bindings.